frontend/components/AgentIntroduction/AgentIntroduction.tsx

```diff
@@ -97,7 +99,7 @@ export const AgentIntroduction = ({
         ))
       }
       renderAgentSelection={renderAgentSelection}
-      styles={styles}
+      styles={{ ...(styles || {}), ...(steps[onboardingStep]?.styles || {}) }}
```

Comment:
@mohandast52 I had to handle it this way, since their pictures have different sizes, but lmk if that's not good (to me it's not :D)

Reply:
b/c each image can have different height, and that's moving the whole content depending on the description height... I will try to handle that better

Reply:
It's a little weird transition on changing the steps 😓 . Can we please ease the animation for height.
https://github.com/user-attachments/assets/3d2f9fb5-0e97-474e-b281-0048682d0be5